test/integration/cli/test/index.test.js

```diff
@@ -760,9 +755,11 @@ describe('CLI Usage', () => {
         }
       )
       try {
-        await check(() => output, /Network:\s*http:\/\/\[::\]:(\d+)/)
-        await check(() => output, /https:\/\/localhost:(\d+)/)
-        await check(() => output, /Certificates created in/)
+        await retry(() => {
+          expect(output).toMatch(/Network:\s*https:\/\//)
```

Comment:
@ztanner This seems fine to me. The original assertion expected `http` for the network URL though so I'm flagging this in case the current behavior is unintended.

Reply:
Talked offline. This is fine.